cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -144,6 +144,28 @@ public KafkaBrokerConfigurationBuilder withCruiseControl(String clusterName, Cru
 
         return this;
     }
+    /**
+     * Configures the Strimzi Metrics Reporter. It is set only if user enabled Strimzi Metrics Reporter.
+     *
+     * @param model     Cruise Control Metrics Reporter configuration
+     *
+     * @return Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withStrimziMetricsReporter(StrimziReporterMetricsModel model)   {
+        if (model != null && model.isEnabled()) {
+            printSectionHeader("Strimzi Metrics Reporter configuration");
+            writer.println("kafka.metrics.reporters=io.strimzi.kafka.metrics.YammerPrometheusMetricsReporter");
+            // ** This is not final.... return to this because we can have other user added plugins, also CC etc. Setting for now so we can manually test!!
+            // Look at withUserConfiguration
+            writer.println("metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter");
+            writer.println("prometheus.metrics.reporter.listener.enable=true");
+            writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:8080");
```

Comment:
```suggestion
            writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:" + StrimziReporterMetricsModel.METRICS_PORT);
```

Reply:
@fvaleri I had             `writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:9094");` 
to get it working but
 `writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:");`
will work too as it binds to all interfaces?

Reply:
You always need a hostname and a port. This is what makes a socket for incoming or outgoing connections. We use `0.0.0.0` as hostname to bind to all interfaces (e.g. eth0, lo0, and so on). Then we use a specific port reserved for metrics (9404). By default the plugin uses 8080, which is not fine for our use case.